__tests__/api/app.test.js

```diff
@@ -312,7 +339,7 @@ describe('Testing app', () => {
 
       test('GET: 400 status when given incorrect comment data (invalid username)', () => {
         return request(app)
-          .post('/api/articles/invalid/comments')
+          .post('/api/articles/1/comments')
```

Comment:
These are great test so far. We have the happy paths with 201 as well as a 400 for missing keys or an empty object. There are a few other cases we probably want to test for: a valid id that doesn't exist e.g 999, a 400 for a non-numeric id e.g. /api/articles/abc/comments, and finally, a 404 for a username that is not in our users database. Many of these tests will be similar to those you wrote for the GET request to this endpoint

Reply:
just a question when try to insert a new comment for a valid article_id but invalid user should return 400 status code instead a 404 for a username that is not in our users database?